src/js/controller/instream-adapter.js

```diff
@@ -403,6 +403,16 @@ const InstreamAdapter = function(_controller, _model, _view, _mediaPool) {
      * @return {void}
      */
     this.skipAd = function(event) {
+        const autoPauseAds = _model.get('autoPause').pauseAds;
```

Comment:
Is there a possibility `_model.get('autoPause')` doesn't return an object?

Reply:
Nope.

Reply:
Nope.